src/modules/menu/components/LayerDescriptionPopup.vue

```diff
@@ -145,8 +152,14 @@ $spacing: 8px;
     }
     table {
         font-size: 100%;
-        width: 100%;
+        width: 50%;
```

Comment:
Would be better to use `width: fit-content` and add a right margin to the `td` element
```
table {
  ...
  width: fit-content;
  td {
    padding-right: 2rem;
  }
}
```
Somehow `margin-right` don't work, but padding does.

This way the column width is dependent on the content and not the window size and if some column require the whole window width they can use it without text wrapping.

Reply:
apparently, margin does not work on table internal elements. I did not think about the padding, as 'fit-content' did not work initially. It seems to work with the padding and the fit content, so I can go with your solution.